.eslintrc.js

```diff
@@ -16,5 +16,6 @@ module.exports = {
   rules: {
     // suppress errors for missing 'import React' in files
     'react/react-in-jsx-scope': 'off',
+    'react/prop-types': 'off',
```

Comment:
What is this ignoring ?

Reply:
I'm ignoring this because eslint want the props to be typed with it's own type and not simply with ts type (https://lightrun.com/solutions/jsx-eslint-eslint-plugin-react-react-props-is-missing-in-props-validation-error/)